.github/workflows/updateVersion.yml

```diff
@@ -17,14 +19,14 @@ jobs:
           sed -i -E "s|(/download/v)[0-9]+(\.[0-9]+){1,3}|\1$NEW_VERSION|g" wwwroot/index.html
       - name: Commit changes
         run: |
-          git config --local user.email "action@github.com"
-          git config --local user.name "GitHub Action"
+          git config --local user.email "turbolay@github.com"
```

Comment:
this is different than the change you made in the doc?

https://github.com/WalletWasabi/WasabiDoc/pull/1861/files#diff-bf641e3fabf18090633a02f26d0490cc4cc3379704b05053812168f2a3890156R25

Reply:
fuck thanks